### ISSUE TITLE
ENOENT: no such file or directory, chmod '/opt/buildhome/repo/node_modules/onfido-sdk-ui/scripts/migrate_locales.js'

### ISSUE THREAD

**Issue by TobbeLundberg** [onfido/onfido-sdk-ui#2131]
After upgrading to v10.0.0 from v9.1.3 we're getting this error in our CI builds

```
12:30:56.102 | npm ERR! code ENOENT
-- | --
12:30:56.102 | npm ERR! syscall chmod
12:30:56.102 | npm ERR! path /opt/buildhome/repo/node_modules/onfido-sdk-ui/scripts/migrate_locales.js
12:30:56.102 | npm ERR! errno -2
12:30:56.102 | npm ERR! enoent ENOENT: no such file or directory, chmod '/opt/buildhome/repo/node_modules/onfido-sdk-ui/scripts/migrate_locales.js'
12:30:56.103 | npm ERR! enoent This is related to npm not being able to find a file.
12:30:56.103 | npm ERR! enoent
```

Downgrading to 9.1.3 again fixes it.

Just now tried v10.0.1 and it has the same problem as v10.0.0

**Comment by warrenseine** [comment]
Hello @TobbeLundberg, thank you for reporting this issue. We indeed introduced a regression. It is fixed and we plan to ship 10.0.2 tomorrow.

**Comment by DannyvanderJagt** [comment]
@TobbeLundberg We released a fix for this in version 10.0.2.

**Comment by TobbeLundberg** [comment]
Just confirmed that this is now working. Thank you.
